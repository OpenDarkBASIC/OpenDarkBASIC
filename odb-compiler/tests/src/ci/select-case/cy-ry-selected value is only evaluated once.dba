select value()
    case 1
        stdout "case 1"
    endcase
    case 2
        stdout "case 2"
    endcase
endselect

end

function value()
    stdout "value()"
    result = 2
endfunction result

