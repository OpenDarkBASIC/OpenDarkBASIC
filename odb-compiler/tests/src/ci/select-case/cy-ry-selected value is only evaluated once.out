"value()\ncase 2\n"

